dev/ory/oathkeeper_rules.yaml

```diff
@@ -10,6 +10,22 @@
     handler: allow
   mutators:
     - handler: noop
+
+- id: allow-clear-cookies-endpoint
+  upstream:
+    url: "http://e2e-tests:4012"
```

Comment:
request going to `<(http|https)>://<[a-zA-Z0-9-.:]+>/auth/clearCookies` won't go to the upstream as I understand it?